stockroom/cli.py

```diff
@@ -4,10 +4,26 @@
 from click_didyoumean import DYMGroup  # type: ignore
 from hangar import Repository
 from rich.progress import Progress
-from stockroom import __version__, external
+from stockroom import __version__, console, external
 from stockroom.core import StockRoom
 from stockroom.keeper import init_repo
-from stockroom.utils import clean_create_column, print_columns_added
+from stockroom.utils import clean_create_column
+
+
+def get_stock_obj(path: Path = Path.cwd(), enable_write=False):
+    try:
+        stock_obj = StockRoom(enable_write=True)
```

Comment:
You are not using the `enable_write` from the function argument
```suggestion
        stock_obj = StockRoom(enable_write=enable_write)
```

Reply:
fixed!